src/components/00-materials/README.md

```diff
@@ -36,7 +36,7 @@ this.render() {
 ## Add Icon
 
 - Move the raw SVG file into `raw-icons`
-- The `npm run build-icons` script will optimize and clean the SVG and export it as a JS-importable string for further usage.
+- The `npm run build` script will optimize and clean the SVG and export it as a JS-importable string for further usage.
```

Comment:
We do have 3 scripts. 2 to optimize and clean icon and image files. the 3rd to make JS strings. I Think we have to change docu for that.

Reply:
As far as I know, npm install does all of that already, right?

Reply:
No. If you add a new svg you have to start 2 scripts manually and check in that result.

Reply:
I deleted this, because down in the contribution everything is explained clearly.